pipeline.jsonc

```diff
@@ -1,10 +1,10 @@
 {
-    "eosio-dot-cdt":
-    {
-        "pipeline-branch": "master",
-        "dependencies": // dependencies to pull for cdt integration tests, by branch, tag, or commit hash
-        {
-            "eosio": "develop"
-        }
+  "eosio-dot-cdt": {
+    "pipeline-branch": "master",
+    // dependencies to pull for cdt integration tests, by branch, tag, or commit hash
+    "dependencies": {
+      "eosio": "develop",
+      "eosio.contracts": "develop"
     }
+  }
```

Comment:
**Optional** - I don't like formatting changes like this because it obscures the line `git` history, which can explain why a value is being used when you do blames or use tools like [git lens](https://gitlens.amod.io). But, if you are going to change the whole file, we should remove the `pipeline-branch` key as it hasn't been used in years.

Reply:
I removed the `pipeline-branch`.